spec/mongoid/criteria/modifiable_spec.rb

```diff
@@ -1683,7 +1687,7 @@
               Person.create_with(attrs).find_or_create_by(query)
             end
 
-            it 'gives the write method args precedence' do
+            it 'gives the find method args precedence' do
```

Comment:
Can we document `create_with`, possibly within `find_or_create_by` section in the tutorial, and note there that attributes in `find_or_create_by` override those in `create_with`?